python/benchmark/gen_data.py

```diff
@@ -164,6 +164,13 @@ def _add_extra_parameters(self) -> None:
             else:
                 help_msg = ""
 
+            # Support multiple biases
+            if name == "bias":
```

Comment:
Why not generate bias randomly like the weights?

Reply:
This is code inherited from before. Do we want random bias then?